sympy/physics/quantum/tensorproduct.py

```diff
@@ -309,19 +311,27 @@ def tensor_product_simp_Mul(e):
         (A*C)x(B*D)
 
     """
-    # TODO: This won't work with Muls that have other composites of
-    # TensorProducts, like an Add, Pow, Commutator, etc.
+    # TODO: This don't work with Muls that have other composites of
```

Comment:
I don't think that this change (won't -> don't) should be done.

Reply:
That was a mistake... sorry about that, should have read the diffs more carefully before committing. Will change.